Watermap cleanup (#94)
ASFHyP3/asf-tools: CHANGELOG.md

and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.3.2]

### Added

> * We now provide an ASF Tools docker image: `ghcr.io/asfhyp3/asf-tools`. To pull the lastest stable release, use the 
>  `latest` tag; to pull the development version, use the `test` tag. 

Reviewer: ```suggestion
* We now provide an ASF Tools docker image: `ghcr.io/asfhyp3/asf-tools`. For usage, see the `asf_tools` [README](asf_tools/README.md).
```

and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.3.2]

### Added
* Dockerfile to build the Hyp3 plugin container
* Hyp3 entrypoint added as `asf_tools/etc/entrypoint.sh`
* `__main__.py` to asf_tools to parse arguments and run watermap

> * `hyplib3>=1.6.8,<2` requirement

Author: ```suggestion
```
